test/form.test.ts

```diff
@@ -730,52 +730,6 @@ test("setErrors", async () => {
   expect(field.error).toEqual("WRONG");
 });
 
-test("setErrors repeating", async () => {
```

Comment:
This test shouldn't be removed. It tests that the bar field in a repeating form can have an error set, which should still be possible.

Reply:
I will put it back